### PR TITLE
*: Support custom repository address of images. #378

### LINKED ISSUE #378: [feature] Support custom repository address of images.
**Is your feature request related to a problem? Please describe.**

When pull the images from a private repository, user need to explicitly specify the address of the repository, usually, all images needs do this step, so provide a unified parameter is necessary.

**Describe the solution you'd like**

Provide a parameter which save the value of the repository address in the cm, or in the environment variable, automatically add prefixed to the image path when pulling the images.

**Describe alternatives you've considered**

**Additional context**

### PR DESCRIPTION
**Pull request by runkecheng**
### What type of PR is this?

feature

### Which issue(s) this PR fixes?

Fixes #378

### What this PR does?

Summary:

- Add parameter `imagePrefix` in Chart
- Add Environment Variable `IMAGE_PREFIX` in the Deployment of Operator
- Automatically add prefix for all images

### CODE REVIEW

File: mysqlcluster/container/container.go [hunk 1]
```diff
@@ -61,7 +63,7 @@ func EnsureContainer(name string, c *mysqlcluster.MysqlCluster) corev1.Container
 
 	return corev1.Container{
 		Name:            ctr.getName(),
-		Image:           ctr.getImage(),
+		Image:           fmt.Sprintf("%s/%s", c.GetPrefixFromEnv(), ctr.getImage()),
```
Comment (acekingke) [inline mysqlcluster/container/container.go:66]: what happened when prefix is empty ？

File: mysqlcluster/mysqlcluster.go [hunk 2]
```diff
@@ -375,3 +376,11 @@ func (c *MysqlCluster) GetKey() client.ObjectKey {
 		Name:      c.Name,
 	}
 }
+
+func (c *MysqlCluster) GetPrefixFromEnv() string {
+	prefix := os.Getenv("IMAGE_PREFIX")
+	if len(prefix) == 0 {
```
Comment (acekingke) [inline mysqlcluster/mysqlcluster.go:382]: Cannot force set `docker io `， should support the   private image repository

**Comment by acekingke** [comment]
LGTM

**Review by andyli029 — APPROVED** [review]
LGTM

**Pull request merged**